tests/keras_tests/exporter_tests/tflite_int8/networks/mobilenetv2_test.py

```diff
@@ -16,6 +16,9 @@
 from keras.applications import MobileNetV2
 import numpy as np
 
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+from model_compression_toolkit.core.tpc_models.default_tpc.v5.tpc_keras import generate_keras_tpc
```

Comment:
Why v5? shouldn;t we use "latest" in tests? (I'm actually not sure, I think I use latest but I don't really no what is expected)

Reply:
I think you'te right